geometry_msgs/Pose2D pose
float64 distance
float64 velocity
float32 move_noise
float64 sense_noise
rfid_reader/TagReading sense
Point[] points
float64 weight
bool direction


